static/src/inline-svgs/illustration/my-digest.svg

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
don't the inline SVGs all get passed through `svgmin` as part of the build process anyway? https://github.com/guardian/frontend/blob/ee86f495a849354c2209afd157c81df0277586b4/Gruntfile.js#L120